src/CodeGenerator/InjectorGenerator.php

```diff
@@ -39,70 +41,55 @@ class InjectorGenerator
     use GeneratorTrait;
 
     private const FACTORY_LIST_TEMPLATE = __DIR__ . '/../../templates/factory-list.template';
-    private const INJECTOR_TEMPLATE = __DIR__ . '/../../templates/injector.template';
-    private const INDENTATION_SPACES = 4;
+    private const INJECTOR_TEMPLATE     = __DIR__ . '/../../templates/injector.template';
+    private const INDENTATION_SPACES    = 4;
 
-    /**
-     * @var ConfigInterface
-     */
+    /** @var ConfigInterface */
     private $config;
 
-    /**
-     * @var DependencyResolverInterface
-     */
-    private $resolver;
-
     /**
      * @deprecated
+     *
      * @var DefinitionInterface
      */
     protected $definition;
 
-    /**
-     * @var string
-     */
+    /** @var string */
     private $namespace;
 
-    /**
-     * @var FactoryGenerator
-     */
+    /** @var FactoryGenerator */
     private $factoryGenerator;
 
-    /**
-     * @var AutoloadGenerator
-     */
+    /** @var AutoloadGenerator */
     private $autoloadGenerator;
 
-    /**
-     * @var LoggerInterface
-     */
+    /** @var LoggerInterface */
     private $logger;
 
     /**
      * Constructs the compiler instance
      *
-     * @param ConfigInterface $config The configuration to compile from
+     * @param ConfigInterface             $config The configuration to compile from
      * @param DependencyResolverInterface $resolver The resolver to utilize
-     * @param string|null $namespace Namespace to use for generated class; defaults
-     *     to Laminas\Di\Generated.
-     * @param LoggerInterface|null $logger An optional logger instance to log failures
-     *     and processed classes.
+     * @param string|null                 $namespace Namespace to use for generated class; defaults
+     *                     to Laminas\Di\Generated.
+     * @param LoggerInterface|null        $logger An optional logger instance to log failures
+     *            and processed classes.
      */
     public function __construct(
         ConfigInterface $config,
         DependencyResolverInterface $resolver,
-        string $namespace = null,
-        LoggerInterface $logger = null
+        ?string $namespace = null,
```

Comment:
Please revert his change as it introduces BC break.

Reply:
those ? are added by cs-fix